### PR TITLE
Feature/spring boot 2.x accumulo 2.1

### PR DESCRIPTION
**Pull request by jwomeara**
Combined spring boot 2.x and accumulo 2.1 changes

### CODE REVIEW

**Review by apmoriarty — DISMISSED** [review]
Most of these changes are diffs from code I wrote and that all looks correct. Good work.

File: docs/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>gov.nsa.datawave</groupId>
         <artifactId>datawave-parent</artifactId>
-        <version>5.1.1-SNAPSHOT</version>
+        <version>3.40.05.2.0-SNAPSHOT</version>
```
Comment (matthpeterson) [inline docs/pom.xml]: looks like a rogue copy and paste here

Comment (matthpeterson) [reply docs/pom.xml]: Everything else in the PR looks good to me.

**Review by matthpeterson — DISMISSED** [review]

**Review by matthpeterson — DISMISSED** [review]

**Review by matthpeterson — DISMISSED** [review]

**Comment by jwomeara** [comment]
> Most of these changes are diffs from code I wrote and that all looks correct. Good work.

@apmoriarty trying to understand this comment.  are you saying that I wiped out your code changes?

**Review by matthpeterson — DISMISSED** [review]

**Comment by apmoriarty** [comment]
> > Most of these changes are diffs from code I wrote and that all looks correct. Good work.
> 
> @apmoriarty trying to understand this comment. are you saying that I wiped out your code changes?

Sorry for the confusion: all of my changes that are part of this merge request look god.

**Review by apmoriarty — DISMISSED** [review]

**Review by matthpeterson — DISMISSED** [review]

**Review by matthpeterson — DISMISSED** [review]

**Pull request merged**